tests/server/api/scheduler_test.go

```diff
@@ -43,6 +42,14 @@ func TestScheduleTestSuite(t *testing.T) {
 	suite.Run(t, new(scheduleTestSuite))
 }
 
+func (suite *scheduleTestSuite) SetupSuite() {
+	suite.NoError(failpoint.Enable("github.com/tikv/pd/pkg/dashboard/adapter/skipDashboardLoop", `return(true)`))
```

Comment:
Why does it fix this problem?

Reply:
https://github.com/tikv/pd/issues/7468#issuecomment-1831199529

Yes, dashboard update pd config, which overwrite the config.

Reply:
But why updating the pd server config can overwrite the scheduler config?